front_end/src/app/(main)/accounts/profile/components/user_info.tsx

```diff
@@ -47,15 +48,25 @@ const UserInfo: FC<UserInfoProps> = ({
   );
   useEffect(() => {
     if (!state?.user) {
+      if (state?.errors?.error_code === "SPAM_DETECTED") {
+        setEditMode(false);
+        alert(
+          "Your account has been deactivated for detected spam. Please contact support@metaculus.com if you believe this was a mistake and we will reactivate your account."
```

Comment:
Leave it at 
"Your account has been deactivated for detected spam. Please contact support@metaculus.com if you believe this was a mistake."
 since we aren't promising to reactivate it.